templates/autoscale-existing-vpc.template

```diff
@@ -0,0 +1,1010 @@
+        },
+        "VPCID": {
+            "Type": "AWS::EC2::VPC::Id",
+            "Description": "The ID of the existing VPC where Fortinet FortiGate Auto Scaling will be deployed. The VPC must have the option 'DNS hostnames' enabled and each of the two Availability Zones in the VPC must have at least 1 public subnet and at least 1 private subnet."
+        },
+        "VPCCIDR": {
+            "Type": "String",
+            "AllowedPattern": "^(([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\\.){3}([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]){1}(\\/([0-9]|[1-2][0-9]|3[0-2]))?$",
+            "ConstraintDescription": "must be a valid CIDR block format.",
+            "Description": "The CIDR block of the selected existing VPC in which FortiGate Auto Scaling will be deployed. This can be found in parentheses in the VPC ID parameter selection."
+        },
+        "PublicSubnet1": {
+            "Type": "AWS::EC2::Subnet::Id",
+            "Description": "The ID of the public subnet 1 located in Availability Zone 1 of the selected existing VPC. The Fortinet FortiGate Auto Scaling instances will be deployed here."
+        },
+        "PublicSubnet2": {
+            "Type": "AWS::EC2::Subnet::Id",
+            "Description": "The ID of the public subnet 1 located in Availability Zone 2 of the selected existing VPC. The Fortinet FortiGate Auto Scaling instances will be deployed here."
+        },
+        "PrivateSubnet1": {
+            "Type": "AWS::EC2::Subnet::Id",
+            "Description": "The ID of the private subnet 1 located in Availability Zone 1 of the selected existing VPC. This subnet will be protected by the FortiGates in the public subnet of the same Availability Zone."
+        },
+        "PrivateSubnet2": {
+            "Type": "AWS::EC2::Subnet::Id",
+            "Description": "The ID of the private subnet 1 located in Availability Zone 2 of the selected existing VPC. This subnet will be protected by the FortiGates in the public subnet of the same Availability Zone."
+        },
+        "PrivateSubnetRouteTable": {
+            "Type": "String",
+            "Description": "The ID of the route table associated with the two private subnets."
+        },
+        "FortiGateInstanceType": {
+            "Type": "String",
+            "Default": "c5.large",
+            "AllowedValues": [
+                "t2.small",
+                "c5.large",
+                "c5.xlarge",
+                "c5.2xlarge",
+                "c5.4xlarge",
+                "c5.9xlarge"
+            ],
+            "ConstraintDescription": "must be a valid EC2 instance type.",
+            "Description": "The instance type for the FortiGates in the Auto Scaling group. There are t2.small and compute-optimized instances such as c4 and c5 available with different vCPU sizes and bandwidths. For more information about instance types, see https://aws.amazon.com/ec2/instance-types/."
```

Comment:
Description mentions c4 instances. AllowedValues does not list any c4 instances.

Reply:
I'll add a full list of supported instance types in a follow change.

Reply:
@JaydenLiang

Reply:
done